doc/plutus-core-spec/cardano/builtins1.tex

```diff
@@ -144,6 +138,37 @@ \subsubsection{Built-in types and type operators}
 the former is part of a built-in type whereas the latter is part of the language
 itself.
 
+\newcommand{\syn}[1]{c_{\mathtt{{#1}}}}
+
+\paragraph{Concrete syntax for $\ty{data}$.}
+The concrete syntax for $\ty{data}$ is given by
+
+\begin{minipage}{0.6\linewidth}
+    \centering
+    \[\begin{array}{rcl}
+    \syn{data} & ::= & \texttt{(Constr} \ \syn{integer} \ \syn{list(data)} \texttt{)}\\
+               &  & \texttt{(Map} \ \syn{list(pair(data,data))} \texttt{)}\\
+               &  & \texttt{(List} \ \syn{list(data)} \texttt{)}\\
+               &  & \texttt{(I} \ \syn{integer} \texttt{)}\\
+               &  & \texttt{(B} \ \syn{bytestring} \texttt{)}\\
+    \end{array}\]
+    \label{fig:data-concrete-syntax}
+\end{minipage}
+
+\noindent We interpret these syntactic constants as elements of $\denote{\ty{data}}$ using
+the various $\inj$ functions defined earlier.  Some valid \texttt{data}
+constants are
+
+\begin{verbatim}
+   (con data (Constr 1 [I 2, B #, Map []]) 
```

Comment:
We need to fix the parser, but in the specification I'd put the one that we are goin to choose. That is:
`(con data (Constr 1 [(I 2),(B #), (Map [])])`.

Reply:
Oh thanks:  I forgot about that.